easypost/webhook.py

```diff
@@ -17,3 +27,28 @@ def update(self, **params) -> "Webhook":
         response, api_key = requestor.request(method=RequestMethod.PATCH, url=url, params=params)
         self.refresh_from(values=response, api_key=api_key)
         return self
+
+    @staticmethod
+    def validate_webhook(webhook_body: Dict[str, Any], headers: Dict[str, Any], webhook_secret: str) -> bool:
+        """Validate a webhook originated from EasyPost by comparing the HMAC header to a shared secret.
+        This function will return True if the webhook originated from EasyPost and False if it did not.
+        """
+        secret_validated = False
+        easypost_signature = headers.get("x-hmac-signature")
+
+        if easypost_signature:
+            secret = binascii.hexlify(bytes(unicodedata.normalize("NFKD", webhook_secret), "utf8"))
```

Comment:
Please break this line up to make it easier to read.

Reply:
Broken up